server.js

```diff
@@ -0,0 +1,16 @@
+var webpack = require('webpack')
+var WebpackDevServer = require('webpack-dev-server')
+var config = require('./webpack.config')
+
+new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  inline: true,
+  historyApiFallback: true,
+  noInfo: true
+}).listen(3000, '0.0.0.0', function(err, result) {
+  if (err) {
+    console.log(err)
+  }
+  console.log('Listening at 0.0.0.0:3000')
+})
```

Comment:
@galarant the webpack dev server was basically only added to get hot reloading to work. With this same setup minus the changes for hot reloading you can just run the django server and run a webpack --watch to watch and build the bundles. If hot reloading components ends up being more trouble than its worth down the line once the components are actually doing something, then I can always get rid of it. I don't have any real opinions on it yet.